organization/components/timeslot/TurfSelection.tsx

```
@@ -0,0 +1,167 @@
+            <h3 className="text-lg font-medium text-gray-800">
+              Timeslot Management for{" "}
+              {turfs.find((t) => t._id === selectedTurfId)?.name}
+            </h3>
+          </div>
+
+          {!showForm ? (
+            <div className="space-y-4">
+              <button
+                onClick={() => setShowForm(true)}
+                className="flex items-center gap-2 px-4 py-2 bg-blue-600 text-white rounded-lg hover:bg-blue-700 transition"
+              >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)